### PR TITLE
lint: fix master eslint error

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | lint: fix master eslint error |
| 🇨🇳 Chinese | lint: fix master eslint error |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ An enterprise-class UI design language and React UI library.
 [fossa-url]: https://app.fossa.io/projects/git%2Bgithub.com%2Fant-design%2Fant-design?ref=badge_shield
 [help-wanted-image]: https://flat.badgen.net/github/label-issues/ant-design/ant-design/help%20wanted/open
 [help-wanted-url]: https://github.com/ant-design/ant-design/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22
-[twitter-image]: https://badgen.net/twitter/follow/antdesignui?style=flat-square
+[twitter-image]: https://img.shields.io/twitter/follow/AntDesignUI.svg?label=Ant%20Design
```
Comment (MadCcc) [inline README.md:31]: 这个是要改的吗

Comment (li-jia-nan) [reply README.md:31]: 之前的链接是坏的，图片加载不出来，我换了一个

Comment (li-jia-nan) [reply README.md:31]: image

新的链接

**Review by MadCcc — APPROVED** [review]

**Comment by li-jia-nan** [comment]
- https://github.com/storybookjs/storybook/issues/20976

**Pull request merged**